src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -23,6 +23,9 @@
   <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" Condition=" '$(CommonTargetsPath)' == '' " />
 
   <ItemGroup>
+    <!-- Enable detection of this project SDK - see https://github.com/microsoft/VSProjectSystem/blob/master/doc/overview/about_project_capabilities.md -->
+    <ProjectCapability Include="BuildTSqlScript" />
```

Comment:
Is this a known capability? Or is this just something you've made up?

Reply:
There is no suitable existing, so I made this up based on [the guidance here](https://github.com/microsoft/VSProjectSystem/blob/master/doc/overview/about_project_capabilities.md)

Reply:
Shouldn't we open an issue on that repo then as well to discuss that capability?

Reply:
https://github.com/microsoft/VSProjectSystem/issues/364

Reply:
I updated the name based on a re-read of the docs

Reply:
With the prefix I think we do not need MS "approval"

Reply:
Given the [discussion here](https://github.com/microsoft/VSProjectSystem/issues/364#issuecomment-1472931004) I believe we are good to go @jmezach